tools/scripts/lib_install/spdlog.sh

```diff
@@ -71,7 +71,7 @@ fi
 cd ${extracted_dir}
 mkdir -p build
 cd build
-cmake -DSPDLOG_FMT_EXTERNAL=ON ..
+cmake ../
```

Comment:
`SPDLOG_FMT_EXTERNAL` flag tells `spdlog` to not use bundled `fmt` library, thus no need to sync the external `fmt` library version.
```suggestion
cmake -DSPDLOG_FMT_EXTERNAL=ON ../
```